torch_np/_wrapper.py

```diff
@@ -158,6 +159,77 @@ def stack(arrays, axis=0, out=None, *, dtype=None, casting="same_kind"):
     )
 
 
+def array_split(ary, indices_or_sections, axis=0):
+    tensor = asarray(ary).get()
+    base = ary if isinstance(ary, ndarray) else None
+    axis = _util.normalize_axis_index(axis, tensor.ndim)
+
+    result = _impl.split_helper(tensor, indices_or_sections, axis)
+
+    return tuple(maybe_set_base(_, base) for _ in result)
+
+
+def split(ary, indices_or_sections, axis=0):
+    tensor = asarray(ary).get()
+    base = ary if isinstance(ary, ndarray) else None
```

Comment:
In general, I think that `base` should be dropped. Too big a hassle for an almost-negligible win. It also makes the wrapper heavier, which is not good.

Reply:
Let me open https://github.com/Quansight-Labs/numpy_pytorch_interop/issues/47 and push the decision to a latter stage.

Reply:
for sure